src/views/open-api-docs-view/utils/get-body-parameter-props.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import { getPropertyDetailPropsFromSchemaObject } from './'
+// Types
+import type { OpenAPIV3 } from 'openapi-types'
+import type { PropertyDetailProps } from '../types'
+
+/**
+ * Given the parameter object from a body parameter,
+ * which we expect to be an object with a schema if it exists,
+ *
+ * Return property detail data.
+ */
+export function getBodyParameterProps(
```

Comment:
Added some utility functions to parse out and group "operations", which form the basis of our API docs view, as well as the property details for the "parameters" within each operation.